README.md

```diff
@@ -0,0 +1,26 @@
+# endpoint-operator
+
+[![CircleCI](https://circleci.com/gh/giantswarm/endpoint-operator.svg?style=svg)](https://circleci.com/gh/giantswarm/endpoint-operator) [![Docker Repository on Quay](https://quay.io/repository/giantswarm/endpoint-operator/status "Docker Repository on Quay")](https://quay.io/repository/giantswarm/endpoint-operator)
```

Comment:
Lets please do it like in other projects and have the badge above the title and using the svg style `shield`.

Reply:
Done.